code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -2774,31 +3657,133 @@ Follow by example and make good judgement based on length which list to include
 		body_parts = list("chest")
 		species_allowed = list("Unathi")
 
-	spelunker
-		name = "Spelunker"
-		icon_state = "spelunker"
-		body_parts = list("l_leg","r_leg","l_arm","r_arm","chest","groin","head")
-		species_allowed = list("Vaurca")
+	una_paw_socks
+		name = "Socks Coloration (Unathi, All)"
+		icon_state = "una_pawsocks"
+		body_parts = list("l_foot","r_foot","l_leg","r_leg","l_arm","r_arm","l_hand","r_hand")
+		species_allowed = list("Unathi")
+
+		chest
+			name = "Socks Coloration (Unathi, Chest)"
+			body_parts = list("chest")
+
+		groin
+			name = "Socks Coloration (Unathi, Groin)"
+			body_parts = list("groin")
+
+		left_arm
+			name = "Socks Coloration (Unathi, Left Arm)"
+			body_parts = list("l_arm")
+
+		right_arm
+			name = "Socks Coloration (Unathi, Right Arm)"
+			body_parts = list("r_arm")
+
+		left_hand
+			name = "Socks Coloration (Unathi, (Left Hand)"
+			body_parts = list("l_hand")
+
+		right_hand
+			name = "Socks Coloration (Unathi, Right Hand)"
+			body_parts = list("r_hand")
+
+		left_leg
+			name = "Socks Coloration (Unathi, Left Leg)"
+			body_parts = list("l_leg")
+
+		right_leg
+			name = "Socks Coloration (Unathi, Right Leg)"
+			body_parts = list("r_leg")
+
+		left_foot
+			name = "Socks Coloration (Unathi, Left Foot)"
+			body_parts = list("l_foot")
+
+		right_foot
+			name = "Socks Coloration (Unathi, Right Foot)"
+			body_parts = list("r_foot")
+
+//Vaurcae specific
 
 	delver
 		name = "Delver"
 		icon_state = "delver"
 		body_parts = list("l_leg","r_leg","l_arm","r_arm","chest","groin","head")
 		species_allowed = list("Vaurca")
 
+	spelunker
+		name = "Spelunker"
+		icon_state = "spelunker"
+		body_parts = list("l_leg","r_leg","l_arm","r_arm","chest","groin","head")
+		species_allowed = list("Vaurca")
+
+//Skrell specific
+
+	skr_arms
+		name = "Skrell Arms (All)"
+		icon_state = "skrell_arms"
+		body_parts = list("l_arm","r_arm","l_hand","r_hand")
+		species_allowed = list("Skrell")
+
+		left_arm
+			name = "Skrell Arms (Left Arm)"
+			body_parts = list("l_arm")
+
+		right_arm
+			name = "Skrell Arms (Right Arm)"
+			body_parts = list("r_arm")
+
+		left_hand
+			name = "Skrell Arms (Left Hand)"
+			body_parts = list("l_hand")
+
+		right_hand
+			name = "Skrell Arms (Right Hand)"
+			body_parts = list("r_hand")
+
+	skr_circles_left
+		name = "Skrell Circles (Left)"
+		icon_state = "skrell_circles_l"
+		body_parts = list("head")
+		species_allowed = list("Skrell")
+
+	skr_circles_right
+		name = "Skrell Circles (Right)"
+		icon_state = "skrell_circles_r"
+		body_parts = list("head")
+		species_allowed = list("Skrell")
+
+	skr_dots
+		name = "Skrell Dots"
+		icon_state = "skrell_dots"
+		body_parts = list("head")
+		species_allowed = list("Skrell")
+
+	skr_heterochromia_left
+		name = "Skrell Heterochromia (Left Eye)"
+		icon_state = "sk_heterochromia_l"
+		body_parts = list("head")
+		species_allowed = list("Skrell")
+
+	skr_heterochromia_right
```

Comment:
I dont know about this one, especially for skrell.

Reply:
> Please note that names may not necessarily dictate what they are actually used for in-universe on someone's character.

![image](https://user-images.githubusercontent.com/3950115/65822998-c84fdd80-e202-11e9-8f85-35788f4c750c.png)

I can rename it to something else if it is trouble, because I am a little curious as to how the reaction would've gone and what you would've been led to think if I had.